carla_integration/carma_docker.launch.py

```diff
@@ -44,18 +44,68 @@ def generate_launch_description():
         name = 'vehicle_config_dir', default_value = '/opt/carma/vehicle/config', description = "Path to vehicle configuration directory"
     )
 
+    #Declare the route file folder launch argument
+    route_file_folder = LaunchConfiguration('route_file_folder')
+    declare_route_file_folder = DeclareLaunchArgument(
+        name = 'route_file_folder',
+        default_value='/opt/carma/routes/',
+        description = 'Path of folder containing routes to load'
+    )
+
+    # Declare enable_guidance_plugin_validate
+    enable_guidance_plugin_validator = LaunchConfiguration('enable_guidance_plugin_validator')
+    declare_enable_guidance_plugin_validator = DeclareLaunchArgument(
+        name = 'enable_guidance_plugin_validator', 
+        default_value='true', 
+        description='Flag indicating whether the Guidance Plugin Validator node will actively validate guidance strategic, tactical, and control plugins'
+    )
+
+    # Declare strategic_plugins_to_validate
+    strategic_plugins_to_validate = LaunchConfiguration('strategic_plugins_to_validate')
+    declare_strategic_plugins_to_validate = DeclareLaunchArgument(
+        name = 'strategic_plugins_to_validate',
+        default_value = '[RouteFollowing]',
+        description = 'List of String: Guidance Strategic Plugins that will be validated by the Guidance Plugin Validator Node if enabled'
+    )
+
+    # Declare tactical_plugins_to_validate
+    tactical_plugins_to_validate = LaunchConfiguration('tactical_plugins_to_validate')
+    declare_tactical_plugins_to_validate = DeclareLaunchArgument(
+        name = 'tactical_plugins_to_validate',
+        default_value='[InLaneCruisingPlugin, StopandWaitPlugin, CooperativeLaneChangePlugin, UnobstructedLaneChangePlugin, YieldPlugin]',
+        description='List of String: Guidance Tactical Plugins that will be validated by the Guidance Plugin Validator Node if enabled'
+    )
+
+    # Declare strategic_plugins_to_validate
```

Comment:
Update to `# Declare control_plugins_to_validate` here and in other carma_docker.launch.py files